bedrock/base/templates/includes/protocol/navigation/index.html

```diff
@@ -24,12 +24,14 @@
         {% if not hide_nav_download_button %}
```

Comment:
Suggest changing this to conveying that it hides both the download and accounts buttons? e.g. `hide_nav_cta`

Reply:
Hi @alexgibson, 

I made your suggestion, that makes more sense. And I didn't realize that two issues could go in the same pull request. Sorry about that. 

Lastly, I saw you guys started a discussion on the #7320, let me know if you need more changes.